doc/tutorials/using_partition_mode_on_up2.rst

```diff
@@ -181,12 +181,19 @@ Enabling partition mode
      shell> fs1:
      fs1:\> EFI\ubuntu\grubx64.efi
 
-  Here we will see the following menu. Select entry ``ACRN Partition Mode`` and
+  Here we will see the following GRUB menu. Select entry ``ACRN Partition Mode`` and
   press ``Enter`` and then the ACRN hypervisor will be loaded automatically.
 
-  .. figure:: images/partition_mode_grub_menu.png
-     :align: center
-     :name: Partition Mode Grub Menu
+  .. code-block:: console
+     :emphasize-lines: 2
+
+      ACRN Partition Mode with Zephyr
```

Comment:
Can you remove this line, it's not added as part of the tutorial, nor is there by default.